src/views/pacbio/PacbioPlateIndex.vue

```diff
@@ -53,11 +55,12 @@
           {{ row.detailsShowing ? 'Hide' : 'Show' }} Plate
         </traction-button>
       </template>
-
       <template #row-details="row">
         <Plate
           v-if="currentPlate.id == row.item.id"
```

Comment:
Should it be row.id instead of row.item.id?

Reply:
@yoldas Thank you for your comments. I will look into it

Reply:
> Should it be row.id instead of row.item.id?
row.item is the the attribute representing the data associated with the row and in this context it is the plate. So we trying the display the plate here when a row is clicked and v-if make sure that the plate id ( which is row.item.id) is same the currentPlate. Hope that makes sense.